pkg/kubelet/token/token_manager.go

```diff
@@ -89,6 +92,8 @@ type Manager struct {
 	// mocked for testing
 	getToken func(name, namespace string, tr *authenticationv1.TokenRequest) (*authenticationv1.TokenRequest, error)
 	clock    clock.Clock
+
+	maxJitter time.Duration
```

Comment:
And delete this field :) Just use the consts.

Reply:
I asked for the field so we can set to 0 in unit tests that don't want jittering

Reply:
ah, that's my initial approach. @liggitt made a point that having it as a variable in manager struct let us disable jittering in the test. https://github.com/kubernetes/kubernetes/pull/96037#discussion_r515241229 makes sense, while personally i prefer doing it lazily by having a global const and only create it in manager struct when we need to disable the jitter unittest.

Reply:
took at look at the unittest, the current test cases have the same effect as disabling jitter because they are minutes granularity. not sure we are going to test jittering here.